pandas/tests/io/formats/test_format.py

```diff
@@ -3432,3 +3432,10 @@ def test_format_remove_leading_space_dataframe(input_array, expected):
     # GH: 24980
     df = pd.DataFrame(input_array).to_string(index=False)
     assert df == expected
+
+
+def test_to_string_complex_number_trims_zeros():
+    s = pd.Series([1.000000 + 1.000000j, 1.0 + 1.0j, 1.05 + 1.0j])
```

Comment:
why should these have 2 decimal zeros and not 1 likely ordinary floats? Where did you get the expected output from?

Reply:
There's a 1.05 in the last element, the expected output is similar to what happens for floats:
```python
[ins] In [2]: s = pd.Series([1.0, 1.0000, 1.05])

[ins] In [3]: s
Out[3]: 
0    1.00
1    1.00
2    1.05
dtype: float64
```